test/test_ops.py

```diff
@@ -0,0 +1,38 @@
+# Copyright Contributors to the Pyro project.
+# SPDX-License-Identifier: Apache-2.0
+
+import weakref
+
+import pytest
+
+from funsor.distribution import BACKEND_TO_DISTRIBUTIONS_BACKEND
+from funsor.util import get_backend
+from funsor.ops import WrappedTransformOp
+
+
+@pytest.fixture
+def dist():
```

Comment:
This seems like a useful pattern, we may want to copy it in the distribution tests in a later PR.